a_tour_of_textile.yml

```diff
@@ -0,0 +1,472 @@
+examples:
+  start_a_chat:
+    add_thread:
+      cmd:
+        code: 'textile threads add "Chat" --type="open" --sharing="shared" --whitelist="P7X3gZus5H15tWCxk4oP6EVsgAM9vwUfCyepAKw49QuRyPYs"'
+      js_http_client:
+        code: 'Textile.threads.add("chat", undefined, undefined, "open", "shared", ["P7X3gZus5H15tWCxk4oP6EVsgAM9vwUfCyepAKw49QuRyPYs"])'
```

Comment:
Are these method static? I though you had to init an object first?

Reply:
right, i was thinking about this and how we wanted convention to be in the docs. init a Textile instance with every code snippet or perhaps the start of any unified example have a section of text that points to the right way to init Textile (for all libraries)

one thing I think probably needed for each line is `await Textile.blah`. yeah?

Reply:
re: `await`, I believe so yeah. Maybe lowercase `textile` as well, otherwise it appears to be a static lib.

just to reach parity with the cmd side, a little node near the top where it talks about needing to have the daemon running would be a good place to insert a link to how to init the js client

Reply:
and then, where it talks about starting another peer, you could have a note initializing another instance of the js client

Reply:
perf, let me try to add that stuff now

Reply:
awesome